toonz/sources/toonzqt/paletteviewergui.cpp

```diff
@@ -1325,14 +1367,11 @@ bool PageViewer::event(QEvent *e) {
       TColorStyle *style = m_page->getStyle(indexInPage);
       if (style) {
         int styleIndex = m_page->getStyleId(indexInPage);
-        toolTip        = "#" + QString::number(styleIndex) + " " +
-                  QString::fromStdWString(style->getName());
-
-        int shortcutKey = m_page->getPalette()->getStyleShortcut(styleIndex);
-        if (shortcutKey > 0)
-          toolTip += QString::fromStdWString(std::wstring(L" (") +
-                                             (wchar_t)shortcutKey + L")");
```

Comment:
Also reverted this back to the original code.  I can do PR's on the last two changes if you wish @shun-iwasawa but it will be a little while unless you deem them necessary.  Thanks.